### PR TITLE
feat(main): KTL-580: create main page

### PR DESCRIPTION
**Pull request by zoobestik**

### CODE REVIEW

File: src/components/MainPage/MainPage.module.css [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+    .intro::after {
+        display: block;
+        content: '';
+        position: absolute;
+        top: 0;
+        right: 0;
+        width: 632px;
+        height: 798px;
+        background: url("./logo.png") no-repeat top center;
```
Comment (krutilov) [inline src/components/MainPage/MainPage.module.css:18]: 2.5mb image seems too big, maybe we can change it to GatsbyImage, or at least compress ?

File: src/components/MainPage/MainPage.module.css [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    }
+}
+
+.companies {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    list-style: none;
+    padding: 0;
+    gap: 16px 94px;
```
Comment (krutilov) [inline src/components/MainPage/MainPage.module.css:29]: A few months ago we decided not to use `flex-gap` due to not great browser support, are we want to use it now?

File: src/pages/faq/index.tsx [hunk 3]
```diff
@@ -24,7 +24,7 @@ export default function FAQPage() {
                         <li className="ktl-offset-bottom-s">Filing issues or suggestions. Please use the{' '}
                             <Link href="https://youtrack.jetbrains.com/issues/KT">Kotlin{'\u00A0'}Issue{'\u00A0'}Tracker</Link> to file issues or suggestions.
                         </li>
-                    </ul>
+                    </ul>to
```
Comment (nikpachoo) [inline src/pages/faq/index.tsx:27]:
```suggestion
                    </ul>
```

File: src/pages/index.tsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+import {Link} from "gatsby";
+
+const SPONSORS = [
+    { name: 'Google', image: googleLogoSvg },
+    { name: 'JetBrains', image: jbLogoSvg },
+]
+
+export default function MainPage() {
+    return <Layout
+        intro={
```
Comment (nikpachoo) [inline src/pages/index.tsx:16]: The Intro block is relevant only for the main page, why not pass it as a child to Layout?
Now, in my opinion, the layout contains logic that might not be there.

**Pull request merged**